lib/tasks/load_us_counties.rake

```diff
@@ -2,13 +2,12 @@ require 'csv'
 namespace :db do
   desc "Load US Counties"
   task :load_us_counties => :environment do
-    if County.none?
-      puts 'loading US counties.  '
+      puts 'loading US counties and equivalents.  '
       counties = []
-      CSV.foreach("lib//assets//us_county_list.csv") do |row|
-        counties<< {name: row[0], region: row[1]}
+      CSV.foreach("lib//assets//us_county_and_equivalent_list.csv") do |row|
+category = row[2] ? row[2].strip.presence : "US_County"
+counties << { name: row[0], region: row[1].strip, category: category }
```

Comment:
Spacing is off here? :) Otherwise looks good!

Reply:
That darn rubocop -- just when you start relying on it!